sea.js

```diff
@@ -168,24 +171,31 @@
 
     if(SEA.window){
       api.crypto = window.crypto || window.msCrypto;
+      if(!api.crypto) {
+        api.crypto = require('isomorphic-webcrypto');
+      }
       api.subtle = (api.crypto||o).subtle || (api.crypto||o).webkitSubtle;
       api.TextEncoder = window.TextEncoder;
-      api.TextDecoder = window.TextDecoder;
+      api.TextDecoder = window.TextDecoder;      
```

Comment:
Trailing white-space... not biggie but easily avoided using editor settings. In my past these caused failing of the coding conventions check...

Reply:
done